examples/_includes/header.xml.njk

```diff
@@ -0,0 +1,11 @@
+<header style="header">
+  {% if hv_button_behavior == 'close' %}
+    <text action="close" href="#" style="header-btn">Close</text>
+  {% elif hv_button_behavior == 'none' %}
+    {# No button is shown #}
+  {% else %}
+    {# defaulting to back button #}
+    <text action="back" href="#" style="header-btn">Back</text>
+  {% endif %}
```

Comment:
I think it would be cleaner to explicitly handle the rendering of the button, and default to no button when prop is not set:
```suggestion
  {% if hv_button_behavior == 'close' %}
    <text action="close" href="#" style="header-btn">Close</text>
  {% elif hv_button_behavior == 'back' %}
    <text action="back" href="#" style="header-btn">Back</text>
  {% endif %}
```